### PR TITLE
Added servlet to get System.getProperty("jboss.node.name")

### PR DESCRIPTION
**Pull request by tommaso-borgato**
Needed for https://issues.jboss.org/browse/EAP7-1074 in order to check whether or not the response comes from one of the nodes listed in the JSESSIONID cookie

### CODE REVIEW

**Comment by rhusar** [comment]
@tommaso-borgato Any reason why not to just add that to the debug servlet? (which contains this information already anyway iirc)

**Comment by tommaso-borgato** [comment]
@rhusar I didn't realize it was already there; it would be a little harder to parse that response and could break the tests if someone does not realize that info is used for a specific purpose, anyway, I can re-use the debug servlet and drop this one

**Comment by rhusar** [comment]
@tommaso-borgato right, we need to make sure that if its used it won't be broken -- which is done by adding tests here :-)

**Comment by tommaso-borgato** [comment]
@rhusar then I shall add a tests for the debug servlet, correct?

**Comment by rhusar** [comment]
@tommaso-borgato I can add the test now, give me a minute

**Comment by rhusar** [comment]
@tommaso-borgato Ok done: https://github.com/clusterbench/clusterbench/pull/76

**Comment by rhusar** [comment]
@tommaso-borgato So, I think the last consensus was that this is desirable, right? Can you fix the commit metadata and rebase?

File: clusterbench-ee8-web/src/main/java/org/jboss/test/clusterbench/info/JBossNodeNameServlet.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package org.jboss.test.clusterbench.info;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@WebServlet(name = "JBossNodeNameServlet", urlPatterns = { "/jboss-node-name" })
```
Comment (rhusar) [inline clusterbench-ee8-web/src/main/java/org/jboss/test/clusterbench/info/JBossNodeNameServlet.java]: `/jboss-node-name` I would drop jboss. Everything we do is 'jboss'.

**Pull request merged**